### PR TITLE
UI: fix failing policy test

### PR DESCRIPTION
**Pull request by hellobontempo**

### CODE REVIEW

File: ui/tests/acceptance/policies/index-test.js [hunk 1]
```diff
@@ -120,8 +121,10 @@ module('Acceptance | policies/acl', function (hooks) {
       'navigates to policy show on successful save'
     );
     assert.dom(SELECT.policyTitle).hasText(policyLower, 'displays the policy name on the show page');
-    // will fail if you have a license about to expire.
```
Comment (hellobontempo) [inline ui/tests/acceptance/policies/index-test.js:123]: It wasn't clear to me how this comment related to whether or not a flash message showed. Acceptance tests tend to rack up the flash messages, so opted to spy + assert for a flash message we _do_ expect
Screenshot 2024-06-10 at 6 47 42 PM

File: ui/tests/acceptance/policies/index-test.js [hunk 2]
```diff
@@ -97,6 +97,7 @@ module('Acceptance | policies/acl', function (hooks) {
   });
 
   test('it can create and delete correctly', async function (assert) {
+    const flashSuccessSpy = sinon.spy(this.owner.lookup('service:flash-messages'), 'success');
```
Comment (hashishaw) [inline ui/tests/acceptance/policies/index-test.js:100]: I've experienced a lot of flakiness using sinon to check flash messages. My latest approach has been to check the contents of `GENERAL.latestFlashContent` selector, which seems to work pretty well. If it's passing consistently locally that might not be necessary, although it's nice to have consistency in how we check these.

**Review by hashishaw — APPROVED** [review]
one non-blocking comment

File: ui/tests/acceptance/policies/index-test.js (hunk 2 above)
Comment (hellobontempo) [reply ui/tests/acceptance/policies/index-test.js:100]: It seemed to past consistently locally, but I can update so we assert these the same!

**Pull request merged**